webapp/src/components/post_type_zoom/index.js

```diff
@@ -13,10 +13,12 @@ import PostTypeZoom from './post_type_zoom.jsx';
 function mapStateToProps(state, ownProps) {
     const post = ownProps.post || {};
     const user = state.entities.users.profiles[post.user_id] || {};
+    const meetingCreator = state.entities.users.profiles[post.props.meeting_creator] || {};
```

Comment:
I haven't experimented with this, but: what happens if a user loads a channel with this post in it, but doesn't have the meeting starter's user profile in their store?

Reply:
My assumption is the username would default to `zoom` because of lines 20 and 21

Reply:
I am not sure I fully understand the issue here, but shouldn't we be attaching the Display Name (and maybe the Username too) of the creator to the meeting post itself? And then generate the UI entirely from the props?

Reply:
@levb I can make the change to pass the username. I feel because a case can exist when a user might not have provided their name, it might be better to pass the username instead.